keras/models.py

```diff
@@ -959,7 +959,7 @@ def fit(self,
                               steps_per_epoch=steps_per_epoch,
                               validation_steps=validation_steps)
 
-    def evaluate(self, x, y, batch_size=32, verbose=1,
+    def evaluate(self, x, y, batch_size=None, verbose=1,
```

Comment:
The docstrings would need to be updated ("Integer. If unspecified, it will default to 32.")

Reply:
docstrings updated